docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-layout.md

```diff
@@ -186,14 +215,18 @@ Configuration example:
 
 ``` xml
 <layout>
-  <complex_key_hashed />
+  <complex_key_hashed>
+    <preallocate>0</preallocate>
+    <shards>1</shards>
+    <!-- <shard_load_queue_backlog>10000</shard_load_queue_backlog> -->
+  </complex_key_hashed>
 </layout>
 ```
 
 or
 
 ``` sql
-LAYOUT(COMPLEX_KEY_HASHED())
+LAYOUT(COMPLEX_KEY_HASHED([PREALLOCATE 0] [SHARDS 1] [PER_SHARD_LOAD_BACKLOG 10000]))
```

Comment:
Need to fix documentation `per_shard_load_queue_backlog` into `shard_load_queue_backlog`.

Reply:
Oh, now I see, thanks! Fixed.